### ISSUE TITLE
Set sidebar autohide animations peed

### ISSUE THREAD

**Issue by MithicSpirit** [black7375/Firefox-UI-Fix#992]
### Is your feature request related to a problem?

The sidebar autohide is too slow, so if I accidentally hover over it, I have to wait too long for it to go away.

### The solution I'd like

I'd like to be able to control the easing function (I'm thinking ease-in cubic or quartic), or at least set the animation speed.

### Alternatives I've considered

Completely disabling the autohide animation would help, but it can be annoying if I'm trying to click on something at the edge of the sidebar. Setting `ui.prefersReducedMotion` to `1` disables that animation entirely, but it also disables other animations like showing or hiding the toolbar altogether.
I suspect that there might already be a way to do what I want to, but I'm not sure how, so maybe it should be added to the [documentation](https://github.com/black7375/Firefox-UI-Fix/wiki/Options).

**Comment by black7375** [comment]
Speed ​​control may be possible.

https://github.com/black7375/Firefox-UI-Fix/blob/c327fb7fe04949990993f6deb218d144e6b364c4/src/sidebar/_overlap.scss#L6-L25

You can edit userChrome.css.
```css
/** Import Lepton stylesheet (Chrome) *****************************************/
@import url("css/leptonChrome.css");

/** Add your custom styles below **********************************************/
:root {
  --uc-autohide-sidebar-speed: 750ms; 
  --uc-autohide-fullscreen-sidebar-speed: 1s; 
  --uc-autohide-sidebar-delay: 600ms; 
}
```

Since the easing function is used globally, it is recommended to override it inside `#sidebar-box`.
https://github.com/black7375/Firefox-UI-Fix/blob/master/src/autohide/sidebar/_static.scss
```css
#sidebar-box {
  --animation-easing-function: cubic-bezier(0.07, 0.95, 0, 1);
}
``

**Comment by MithicSpirit** [comment]
Works perfectly, thanks! (And with the -delay option, I don't even really need the easing function.)
